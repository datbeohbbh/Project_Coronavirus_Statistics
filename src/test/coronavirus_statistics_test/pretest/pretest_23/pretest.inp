26 21
? 5
? 5
q 1 26 291
q 1 1 270
q 1 12 458
q 1 3 347
q 2 1 187
q 3 12 413
q 2 3 247
? 4 4
? 7
? 6
q 1 22 219
? 7
q 1 24 236
? 6
? 4 5
? 4 5
? 4 1
? 7
? 6