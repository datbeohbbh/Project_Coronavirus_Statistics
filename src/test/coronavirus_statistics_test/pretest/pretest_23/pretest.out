Query ? 5 :
The number of safe cities is 26

Query ? 5 :
The number of safe cities is 26

Query ? 4 4 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 291
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 413 people

Query ? 6 :
The city with the highest recovery is city with the index 3
Quantity: 247 people

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 413 people

Query ? 6 :
The city with the highest recovery is city with the index 3
Quantity: 247 people

Query ? 4 5 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 236
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 24 
- Information of the city 24 -
-> Number of current patients : 236
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 45
-> Has recovered : 0
-> Was dead : 413

Query ? 7 :
The city with the highest death is city with the index 12
Quantity: 413 people

Query ? 6 :
The city with the highest recovery is city with the index 3
Quantity: 247 people

